users/tests/test_unit_users_apiview.py

```diff
@@ -122,7 +122,12 @@ def test_user_create_view_should_display_create_user_form_on_get(self):
 
     def test_user_create_view_should_add_new_user_on_post(self):
         request = APIRequestFactory().post(
-            path=reverse("custom-user-create"), data={"email": "anothertestuser@codepoets.it"}
+            path=reverse("custom-user-create"),
```

Comment:
We agreed that we will move refactored tests to `*/tests/test_views.py` to keep track which ones are not refactored yet, also name is fitting better :)

Reply:
Also it would be great if you could refactor them to use `self.client`, as `APIRequestFactory` is rather for rest framework views testing. You can find all examples in already refactored tests.